woodwork/accessor_utils.py

```diff
@@ -30,14 +30,16 @@ def init_series(series, logical_type=None, semantic_tags=None,
     Returns:
         Series: A series with Woodwork typing information initialized
     """
-    logical_type = _get_column_logical_type(series, logical_type, series.name)
-
-    new_series = logical_type.transform(series)
-    new_series.ww.init(logical_type=logical_type,
-                       semantic_tags=semantic_tags,
-                       use_standard_tags=use_standard_tags,
-                       description=description,
-                       metadata=metadata)
+    if _is_series(series):
+        logical_type = _get_column_logical_type(series, logical_type, series.name)
+        new_series = logical_type.transform(series)
+        new_series.ww.init(logical_type=logical_type,
+                           semantic_tags=semantic_tags,
+                           use_standard_tags=use_standard_tags,
+                           description=description,
+                           metadata=metadata)
+    else:
+        raise TypeError(f'Input must be of series type. The current input is of type {type(series)}')
```

Comment:
This is a minor point, but we could refactor like this:
```
if not _is_series(series):
    raise TypeError(f'Input must be of series type. The current input is of type {type(series)}')

logical_type = _get_column_logical_type(series, logical_type, series.name)
new_series = logical_type.transform(series)
new_series.ww.init(logical_type=logical_type,
                   semantic_tags=semantic_tags,
                   use_standard_tags=use_standard_tags,
                   description=description,
                   metadata=metadata)
```
This keeps all the normal initialization stuff outside of the if/else block and (I think) makes it a little more clear that we are raising the TypeError when the input is not a series, as it puts the `raise` right by the if clause.

Reply:
Alright yeah that sounds good, I have refactored it as such